added advice how to reference existing lambda functions (#8226)
aws-amplify/docs: src/pages/[platform]/build-a-backend/data/custom-business-logic/index.mdx

});
```

> If you want to use an existing Lambda function, you can reference it by its name: `a.handler.function('name-of-existing-lambda-fn')`. This references an external lambda resource which Amplify is not aware of. You need to make sure the function and all dependencies are being managed outside of Amplify.

Author: ```suggestion
If you want to use an existing Lambda function, you can reference it by its name: `a.handler.function('name-of-existing-lambda-fn')`. Note that Amplify will not update this external Lambda function or its dependencies.
```
